package.json

```diff
@@ -12,61 +12,66 @@
   },
   "scripts": {
     "build": "ember build --environment=production",
-    "lint": "npm-run-all --aggregate-output --continue-on-error --parallel \"lint:!(fix)\"",
+    "lint": "concurrently \"npm:lint:*(!fix)\" --names \"lint:\"",
     "lint:dependency": "ember dependency-lint",
-    "lint:fix": "npm-run-all --aggregate-output --continue-on-error --parallel lint:*:fix",
+    "lint:css": "stylelint \"**/*.css\"",
+    "lint:css:fix": "concurrently \"npm:lint:css -- --fix\"",
+    "lint:fix": "concurrently \"npm:lint:*:fix\" --names \"fix:\"",
     "lint:hbs": "ember-template-lint .",
     "lint:hbs:fix": "ember-template-lint . --fix",
     "lint:js": "eslint . --cache",
     "lint:js:fix": "eslint . --fix",
     "start": "ember serve",
     "start:local": "cross-env LOCAL_API=true ember s",
-    "test": "percy exec -- ember test"
+    "test": "concurrently \"npm:lint\" \"npm:test:*\" --names \"lint,test:\"",
+    "test:ember": "ember test"
   },
   "devDependencies": {
+    "@babel/eslint-parser": "^7.21.3",
+    "@babel/plugin-proposal-decorators": "^7.21.0",
     "@ember/optional-features": "^2.0.0",
+    "@ember/string": "^3.0.1",
     "@ember/test-helpers": "^2.9.3",
     "@glimmer/component": "^1.1.2",
     "@glimmer/tracking": "^1.1.2",
     "@percy/cli": "^1.0.0-beta.65",
     "@percy/ember": "^3.0.0",
     "@sinonjs/fake-timers": "^7.1.2",
-    "babel-eslint": "^10.1.0",
     "broccoli-asset-rev": "^3.0.0",
+    "concurrently": "^8.0.1",
     "cross-env": "^7.0.3",
     "dayjs": "^1.10.6",
-    "ember-auto-import": "^2.1.0",
-    "ember-cli": "~4.0.1",
+    "ember-auto-import": "^2.6.3",
+    "ember-cli": "~4.12.3",
     "ember-cli-app-version": "^6.0.0",
     "ember-cli-babel": "^7.26.11",
     "ember-cli-dependency-checker": "^3.3.1",
     "ember-cli-dependency-lint": "^2.0.1",
-    "ember-cli-deploy": "^1.0.0",
     "ember-cli-deprecation-workflow": "^2.1.0",
     "ember-cli-htmlbars": "^6.2.0",
     "ember-cli-inject-live-reload": "^2.1.0",
-    "ember-cli-mirage": "^2.4.0",
+    "ember-cli-mirage": "^3.0.2",
     "ember-cli-netlify": "^0.4.1",
     "ember-cli-sri": "^2.1.1",
     "ember-cli-terser": "^4.0.2",
     "ember-css-modules": "^1.5.0",
-    "ember-data": "~4.0.2",
-    "ember-export-application-global": "^2.0.1",
+    "ember-data": "~4.11.3",
```

Comment:
I would update that to EmberData 4.12. This was a mistake in the blueprint because EmberData v4.12 wasn't released when the ember-cli release went out. But it's out now, so we can update to the latest v4 release.

Reply:
Yeah makes sense, Thanks you. Updating to 4.12